build/stf-run-ci/README.md

```diff
@@ -53,7 +53,7 @@ choose to override:
 | `__service_telemetry_trap_default_oid`                 | <default_oid>            | 1.3.6.1.4.1.50495.15.1.2.1                            | The trap OID if none is found in the Prometheus alert labels.                                                       |
 | `__service_telemetry_trap_default_severity`            | <default_severity>       | <undefined>                                           | The trap severity if none is found in the Prometheus alert labels.                                                  |
 | `__service_telemetry_logs_enabled`                     | {true,false}             | false                                                 | Whether to enable logs support in ServiceTelemetry                                                                  |
-| `__service_telemetry_observability_strategy`           | <observability_strategy> | `use_community`                                       | Which observability strategy to use for deployment. Default deployment is 'use_community'. Also supported is 'none' |
+| `__service_telemetry_observability_strategy`           | <observability_strategy> | `use_community`                                       | Which observability strategy to use for deployment. Default deployment is 'use_community'. Also supported are 'use_redhat', 'use_hybrid', and 'none' |
```

Comment:
You could update this documentation to match the new defaults which seem to be use_hybrid going forward right?

Reply:
Default in stf-run-ci is defaulting to use_hybrid (mid-term solution I guess to make sure we're covering our non-Red Hat scoped areas which will be migrated ideally in OCP 4.14+), so I think that does look to be the value here vs use_redhat which is the default for new STF installations going forward.